.github/workflows/integration.yml

```
@@ -166,6 +166,9 @@ jobs:
         k8s:
           - v1.23
           - v1.32
+        linkerd-managed-gw-crds:
+          - true
+          - false
```

I've put up https://github.com/linkerd/linkerd2/pull/13891 to support varying test builds by feature (i.e. with `--no-default-features`, only stable tests will be run; and all tests are run with `--no-default-features --features=gateway-api-experimental`).

This could enable us to do something like:

```suggestion
        gateway-api:
          - version: linkerd # v1.1.1-experimental
            channel: experimental
          - version: v1.2.1
            channel: stable
        exclude:
          k8s: v1.23
          gateway-api:
            version: v1.2.1
            channel: stable
```

so that we get the combinations:

```yaml
- k8s: v1.23
  gateway-api:
    version: linkerd
    channel: experimental
- k8s: v1.32
  gateway-api:
    version: linkerd
    channel: experimental
- k8s: v1.32
  gateway-api:
    version: v1.2.1
    channel: stable
```

We would use gateway-api.channel=experimental to indicate that the gateway-api-experimental feature should be enabled and to vary the URL that we fetch.

---

I tried this out in this branch https://github.com/linkerd/linkerd2/pull/13893